db/migrate/20170623163309_replace_join_table_with_assignment_file_in_template_divisions.rb

```diff
@@ -1,8 +1,8 @@
 class ReplaceJoinTableWithAssignmentFileInTemplateDivisions < ActiveRecord::Migration
   def change
     # Remove criteria_assignment_files_join column from TemplateDivision
-    remove_reference :template_divisions, :criteria_assignment_files_join, foreign_key: true
+    remove_reference :template_divisions, :criteria_assignment_files_join
```

Comment:
Don't change migration files after they've happened. Also the foreign key is appropriate here. If there's an error elsewhere, you need to fix the problem there, not by changing the schema.